repository_service_tuf/cli/admin/__init__.py

```diff
@@ -13,11 +13,22 @@
 from repository_service_tuf.cli import click, rstuf
 
 
-def _set_settings(context: click.Context, api_server: Optional[str]):
+def _set_settings(
+    context: click.Context, api_server: Optional[str], headers: Optional[str]
+):
     """Set context.obj['settings'] attributes."""
     settings = context.obj["settings"]
     if api_server:
         settings.SERVER = api_server
+    if headers:
+        settings.HEADERS = dict(
+            (key.strip(), value.strip())
+            for key, value in (
+                header.split(":", 1) for header in headers.split(",")
```

Comment:
Should be a bit more defensive, I think. What happens if the format is even accidentally incorrect?

Reply:
I think we might want to use [HTTP Structured Fields](https://www.fastly.com/blog/improve-http-structured-headers) instead, especially with [this library](https://github.com/mnot/http-sf).

Reply:
Good catch @trishankatdatadog 
However, I had a bad time using the parser.
I handle the error and catch the exception.

nit: I also like the format of `curl -H`.